roles/tpa_single_node/templates/manifests/spog/api/Deployment.yaml.j2

```diff
@@ -48,26 +49,21 @@ spec:
             - -p
             - '8080'
             - --bombastic-url
-            - https://bombastic-api.trustificcation.svc.cluster.local
+            - http://bombastic-api-pod:{{ tpa_single_node_bombastic_api_port }}
             - --vexination-url
-            - https://vexination-api.trustificcation.svc.cluster.local
+            - http://vexination-api-pod:{{ tpa_single_node_vexination_api_port }}
             - --collectorist-url
-            - https://collectorist-api.trustificcation.svc.cluster.local
+            - http://collectorist-api-pod:8080
             - --v11y-url
-            - https://v11y-api.trustificcation.svc.cluster.local
-            - --exhort-url
-            - https://exhort-api.trustificcation.svc.cluster.local
+            - http://v11y-api-pod:{{ tpa_single_node_v11y_api_port }}
             - --crda-url
             - https://rhda.rhcloud.com/api/v4/analysis
             - --guac
-            - https://guac-graphql.trustificcation.svc.cluster.local/query
+            - http://guac-graphql-pod:8080/query
```

Comment:
@gildub this port is configurable in another PR ?

Reply:
@desmax74 , are you asking if this could be done in a separate PR ?